### PR TITLE
Create a PHP 5.5 Magento1 EOL image for older Magento versions not patched

### PR DESCRIPTION
**Pull request by andytson-inviqa**

### CODE REVIEW

File: continuous-pipe.yml [hunk 1]
```diff
@@ -56,6 +56,13 @@ tasks:
             - name: PHP_VERSION
               value: '7.0'
         php56_nginx:
```
Comment (kierenevans) [inline continuous-pipe.yml]: This ID would need to change

File: continuous-pipe.yml [hunk 2]
```diff
@@ -55,6 +55,13 @@ tasks:
           environment:
             - name: PHP_VERSION
               value: '7.0'
+        php55_nginx:
+          image: quay.io/continuouspipe/eol-php5.5-nginx
```
Comment (kierenevans) [inline continuous-pipe.yml]: Saved some time and I've created this and the other one 👍

File: continuous-pipe.yml (hunk 1 above)
Comment (andytson-inviqa) [inline continuous-pipe.yml]: fixed

File: continuous-pipe.yml (hunk 2 above)
Comment (andytson-inviqa) [inline continuous-pipe.yml]: thanks

**Pull request merged**